spec/features/bets/scoreboard_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+feature 'Scoreboard' do
```

Comment:
Ok. Probably we misunderstood each other when we talked about that.
- we don't need three users, two will be enough
- our goal here is also to test sorting, so we need to:
 - leave first part as it is - so user 1 is first after the first round
 - add second part (matches & bets & scores) after which user2 should have more points then user 1 and should be on the top of the table
- change names to User1, User2

Reply:
I'm added new test in ImportMatcher. I don't want test this thing in scoreboard so i don't added here.

Reply:
Still, the refactoring ideas above should be implemented there.